src/App/Patterns/content/Forms/index.js

```diff
@@ -229,30 +229,30 @@ const DeveloperDocumentation = () => (
             <b className="brand-headline">Client-side validation</b>
             <p>Use client-side validation to validate as much data as possible before the user submits the form. Common errors include empty required fields, invalid characters or incorrect input format. To give users inline validation feedback insert a error message within the <CodeTags type="primary" code="form-group"/> in the <CodeTags type="primary" code="help-block"/> element in the attribute <CodeTags type="primary" code="data-error"/>.</p>
 
-            <ComponentPreview language="html" codeFigure showCasePanel removeOuterTag>
+            <EditableComponentPreview language="html" codeFigure showCasePanel removeOuterTag>
```

Comment:
I think it is enough to have only editable component preview in the most upper example. So here we can use the old ComponentPreview, or a view-only version of the Editable one

Reply:
Same for line 241

Reply:
And 280, and 292 and 307 🏆

Reply:
done, switched all to staticPreview (setting code editor as read-only, and output is actual component directly, not through iframe) 👍

Reply:
Hm, they are still editable right? 🤔